Leave/leave_interface.py

```diff
@@ -57,20 +56,24 @@ async def HandleLeaveReactions(client, payload):
     if status == None:
         return
 
-    if utils.isNotBot(payload.member) and utils.IsAdmin(payload.member) and leave_db.IsLeaveRequestPending(payload.message_id):
+    if not (utils.isNotBot(payload.member) and utils.IsAdmin(payload.member)):
+        return
+        
+    if status == "Reverted":
+        if not (leave_db.IsLeaveRequestPending(payload.message_id)):
+            leave_db.UpdateLeaveStatus(payload.message_id, "Pending")
+            await UI.UpdateLeaveEmbed(payload.member, message, embed, "Pending", True)
```

Comment:
I think we'd want to remove the revert reaction here instead of leaving it there like the others. Speaking of, I think we'd want to remove all the other reactions too. Sort of reverting it to its original state. We do lose some information in who approved and who reverted, but those should be done properly in an auditing system later on—not incidentally by the leftover reactions. 

Can you create a card for auditing in to do please?

Reply:
https://trello.com/c/onISUooH/724-discord-bot-action-auditing-for-leaves

Reply:
Done